vcl_templates/www.vcl.erb

```diff
@@ -40,6 +40,55 @@ backend F_origin {
 <% end -%>
 }
 
+<%- if config['replatforming_traffic_split'] %>
+backend F_eks_origin {
+    .connect_timeout = 5s;
+    .dynamic = true;
+    .port = "443";
+    .host = "www-origin.eks.staging.govuk.digital";
+    .first_byte_timeout = 15s;
+    .max_connections = 200;
+    .between_bytes_timeout = 10s;
+    .share_key = "<%= config.fetch('service_id') %>";
+
+    .ssl = true;
+    .ssl_check_cert = <%= config['disable_tls_validation'] ? 'never' : 'always' %>;
+    .min_tls_version = "<%= config.fetch('min_tls_version', '1.2') %>";
+    <%- if config['ssl_ciphers'] -%>
+    .ssl_ciphers = "<%= config['ssl_ciphers'] -%>";
+    <%- end -%>
+    .ssl_cert_hostname = "www-origin.eks.staging.govuk.digital";
+    .ssl_sni_hostname = "www-origin.eks.staging.govuk.digital";
+
+<% if config['probe'] -%>
+    .probe = {
+        .request =
+            "HEAD <%= config['probe'] %> HTTP/1.1"
+            "Host: <%= config.fetch('origin_hostname') %>"
+            "User-Agent: Fastly healthcheck (git version: <%= config['git_version'] %>)"
+<% if config['rate_limit_token'] -%>
+            "Rate-Limit-Token: <%= config['rate_limit_token'] %>"
+<% end -%>
+<% if config['basic_authentication'] -%>
+            "Authorization: Basic <%= config['basic_authentication'] %>"
+<% end -%>
+            "Connection: close";
+        .threshold = 1;
+        .window = 2;
+        .timeout = 5s;
+        .initial = 1;
+        .expected_response = 200;
+        .interval = 10s;
+    }
+<% end -%>
```

Comment:
Maybe this end should be on the next line, have you tried a manual general of the template where this is activated?

Reply:
That entire block was copied from here: https://github.com/alphagov/govuk-cdn-config/blob/0368e27e7b7bbcbd3c31766a55e88dd81b32d404/vcl_templates/www.vcl.erb#L20-L40

It looked good when I generated the template locally too.